docs/_static/nidcpower_usage.inc

```diff
@@ -8,7 +8,7 @@ The following is a basic example of using the **nidcpower** module to open a ses
     import nidcpower
     # Configure the session.
 
-    with nidcpower.Session('dev1') as session:
+    with nidcpower.Session(resource_name='PXI1Slot2', channels='0') as session:
```

Comment:
`channels=0` would have worked here (without the quotes). Isn't this what we are encouraging clients to use?